sites/partners/layouts/index.tsx

```diff
@@ -40,7 +40,19 @@ const Layout = (props) => {
         <main>{props.children}</main>
 
         <SiteFooter>
-          <FooterNav copyright="© 2020 • All Rights Reserved">
+          <FooterSection>
```

Comment:
Would it make sense to pull this out into a shared component? It seems (at first glance) similar to the footer on the public site.

Reply:
We can make them a shared component, but I'm not yet sure if the footers will end up being different between the partner portal and applicant site. This is just my initial guess at adding a footer here. I will keep it as is until I'm sure they should be a shared component.